src/components/YourCompany/BusinessContactInfo.vue

```diff
@@ -27,7 +27,7 @@
         <div id="lbl-phone" v-else>(Not entered)</div>
       </v-flex>
 
-      <v-flex xs1 class="mt-n2">
+      <v-flex xs1 class="mt-n2" v-if="!isCorrection">
```

Comment:
Ticket #5140 !

Reply:
Hmm, actually, should this be rendered for Alteration?

Or for neither atm? If so then I suggest changing this to `v-if="false"` + a comment.

Reply:
I think we will allow this in alteration or profile view, right?

Reply:
That was my assumption at least. If we're never going to allow editing, we can remove all the functionality in the component. I figured it was safer to leave it all in for now, until we know all the requirements around this for the latter filings.

Reply:
ok